field_utils/CMakeLists.txt

```diff
@@ -7,6 +7,11 @@ set(srcs
   FieldUtilities.F90
   FieldUnaryFunctions.F90
   FieldBinaryOperations.F90
+  FieldUnits.F90
+  udunits2.F90
+  udunits2interfaces.F90
+  udunits2encoding.F90
+  udunits2status.F90
```

Comment:
I am not really sure where I want these to live.   But we can move after we get this change merged.

Reply:
OK